### DISCUSSION TITLE
Change Time and Date format in Webmail

### DISCUSSION THREAD

**Discussion by H-A-L-I-S** [modoboa/modoboa#2254] [Q&A] [answered]
I just could not find any option to change the time and date format in modoboa-webmail. 

Help is very appreciated, thank you in advance!

**Comment by H-A-L-I-S** [comment, marked as answer]
Found that editing the language also changes the date/time format. Not really ideal, but better than a.m. and p.m. time :D

**Reply by tonioo** [reply]
Yes, date/time display depends on the current locale.
